# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/cache/QuotaClient.cpp
# Commit: 868577854548
# Full Hash: 868577854548c7442c15ab9b1ad1213af187e01e
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2020-11-17 21:55:29
# Regressor Bug: 1671369
# File Overlap Count: 1
# Description:
#   Bug 1671369 - Change GetUsageForClient to return a UsageInfo. r=dom-workers-and-storage-reviewers,ttung
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D93777
# ==============================================================================

diff -r 5f347d3846aa -r 868577854548 dom/cache/QuotaClient.cpp
--- a/dom/cache/QuotaClient.cpp	Tue Nov 17 12:13:05 2020 +0000
+++ b/dom/cache/QuotaClient.cpp	Tue Nov 17 13:17:24 2020 +0000
@@ -384,13 +384,8 @@
       }()));
 
   if (!maybePaddingSize) {
-    uint64_t usage;
-    if (qm->GetUsageForClient(PERSISTENCE_TYPE_DEFAULT, aGroupAndOrigin,
-                              Client::DOMCACHE, usage)) {
-      return UsageInfo{DatabaseUsageType(Some(usage))};
-    }
-
-    return UsageInfo{};
+    return qm->GetUsageForClient(PERSISTENCE_TYPE_DEFAULT, aGroupAndOrigin,
+                                 Client::DOMCACHE);
   }
 
   CACHE_TRY_INSPECT(